OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -27,11 +44,16 @@
     @InjectView(R.id.auth_webview)
     private AuthenticatedWebView authWebView;
 
+    @InjectView(R.id.info_banner)
+    private LinearLayout infoBanner;
+
     @InjectView(R.id.swipe_container)
     protected SwipeRefreshLayout swipeContainer;
 
+    private CourseDateViewModel courseDateViewModel;
     private PreLoadingListener preloadingListener;
     private boolean isPageLoading = false;
+    private boolean canFetchBannerInfo = true;
```

Comment:
no need to add an extra member, can be removed.

Reply:
removed